inputters/xml.lua

```diff
@@ -70,19 +70,19 @@ function inputter.appropriate (round, filename, doc)
 end
 
 function inputter.parse (_, doc)
-  local tree, err = parse(doc)
-  if not tree then
+  local ast, err = parse(doc)
+  if not ast then
     SU.error(err)
   end
   -- XML documents can have any root element, and it should be up to the class
   -- to supply handling far whatever that element that is in a specific format.
   -- Hence we wrap the actual DOM in an extra element of our own if and only if
   -- it doesn't look like a native SILE one already.
-  local root = tree.command
-  if root ~= "sile" and root ~= "document" then
-    tree = { tree, command = "document" }
+  local rootelement = ast.command
+  if rootelement ~= "sile" and rootelement ~= "document" then
+    ast = { ast, command = "document" }
   end
-  return { tree }
+  return ast
```

Comment:
So before there was one additional table level (`{ xxx }` vs. `xxx`) now.
Do we have any idea why?

Reply:
Last night I didn't have any idea why. Today I remembered it was prep work for a future where we could process multiple documents in one file. Also the Lua inputter is already capable of using this, the XML and SIL inputters just have the extra wrapper as future proofing and don't make use of it yet.

I should not have removed it in this PR.